integration-tests/Tests/IntegrationTests/PrimitiveTests.swift

```diff
@@ -0,0 +1,28 @@
+import RustLib
```

Comment:
Let's add a `GitHub` actions job that runs these `integration-tests`.

Reply:
Good idea.